### ISSUE TITLE
Long click open link doesn't work 

### ISSUE THREAD

**Issue by jabroniw** [Infocatcher/Right_Links#19]
Since the latest Firefox update(37.0.2) long left click to open a link doesn't work

**Comment by Infocatcher** [comment]
Strange, still works fine for me.
Please ensure, that yuu have checked
Options – Enable long left-click
(or _extensions.rightlinks.enabled.left_ in about:config)

**Comment by jabroniw** [comment]
Yes, it is enabled but still doesn't work. I've removed and reinstalled but
still not working. Currently running Firefox 37.0.2

On Wed, Apr 29, 2015 at 7:15 AM, Infocatcher <notifications@github.com>
wrote:

> Strange, still works fine for me.
> Please ensure, that yuu have checked
> Options – Enable long left-click
> (or *extensions.rightlinks.enabled.left* in about:config)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Infocatcher/Right_Links/issues/19#issuecomment-97391169>
> .
>



-- 
Jay Woodford
Coordinator
Schuylkill County Animal Response Team
"No Animal Left Behind"

**Comment by Infocatcher** [comment]
May be some conflict...
Please try on new Firefox profile: https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles
If works fine, this is conflict with some another extension.

Also please provide information from about:support page.
